client/components/pages/files/FilesTable.vue

```diff
@@ -154,7 +173,12 @@ function onRowDoubleClick(event: DataTableRowDoubleClickEvent) {
       </template>
     </PColumn>
 
-    <PColumn field="name" sortable :header="t('pages.files.table.name')">
+    <PColumn
+      style="max-width: 30em; text-overflow: ellipsis; overflow: hidden;"
```

Comment:
my man, we have u n o c s s (avoid the style property when possible to avoid issues with precedence later)